### DISCUSSION TITLE
Can i access variables defined in yaml from dart?

### DISCUSSION THREAD

**Discussion by Nebula26** [AngeloAvv/flutter_flavorizr#221] [Q&A]
Hi,
there are any api to read variables from dart?
For example:
  ios:
      bundleId: "com.example.apple"
      variables:
        VARIABLE_ONE:
          value: "variable1"
          
can i read "VARIABLE_ONE" in my main_banana.dart?

Thanks

**Comment by AngeloAvv** [comment]
Hey @Nebula26 , unfortunately no (using flutter_flavorizr) but you can define them after running the flavorization process right into the F class
